### ISSUE TITLE
Add custom icons on chart(earnings, conference calls, etc.) (v0.4)

### ISSUE THREAD

**Issue by XmelesX** [rrag/react-stockcharts#54]
Only if this possible.
There are news on US market(earnings, conference calls, etc.)
Show you chart for example:
![screenshot_1](https://cloud.githubusercontent.com/assets/16750139/12713590/983d0ea4-c8e0-11e5-9da7-64a41ac93fbd.png)

On hover I get information about news:
![on hover](https://cloud.githubusercontent.com/assets/16750139/12713634/d8b8f236-c8e0-11e5-8d61-f1c5e80e2138.png)

Usually configurations:
**Earning:** 
1) date, format 28-10-2015-15-15 ---> day-month-year-hours(24)-minutes
2) estimates value, format $0.238 prefix is currency symbol for other markets ($, €, £, ¥ etc.)
3) current value, format $0.31 prefix is currency symbol for other markets ($, €, £, ¥ etc.)
4) icon style.

**Conference calls:**
1) date, format 28-10-2015-15-15 ---> day-month-year-hours(24)-minutes
2) icon style.

**Custom news:**
1) date, format 01-01-2016-00-00 ---> day-month-year-hours(24)-minutes
2) event: "Happy New Year"
3) icon style.

For icons will be the best suited font:
https://fortawesome.github.io/Font-Awesome/
https://icomoon.io/app/#/select
or other...

**Comment by rrag** [comment]
this will be a good addition. There is still lot of work left on 0.4, and I don't want to delay it further. will plan 0.5.

**Comment by rrag** [comment]
How does this api look?

```jsx
<Chart>
	...
	...
	<Annotation when={d => d.high - d.low > 1}>
		{props => <Label className="glyphicon" key={props.d.idx} {...props}>&#xe182;</Label>}
	</Annotation>
</Chart>
```

`&#xe182;` is the character which creates the bootstrap icon `glyphicon-earphone`

I got this to work for 

```jsx
<ChartCanvas type="svg" ...>
```

working on a canvas implementation.

**Comment by rrag** [comment]
Changed [API](https://github.com/rrag/react-stockcharts/blob/cdf4551f81aef23584c40c38b5fcc68d6d89a6d8/docs/lib/charts/CandleStickChartWithAnnotation.jsx#L81-L82)

also [example](http://rrag.github.io/react-stockcharts-next/documentation.html#/annotations) updated for both svg and canvas

**Comment by rrag** [comment]
fixed, published in 0.5
